README.md

```
@@ -1,36 +1,35 @@
-Example for user githubuser working on Exercise 1:
+You can also run the project via Docker. To build the Docker image, execute the following commands from the
+directory where this file resides. Please ensure you have installed
+a [Java SE implementation](https://adoptium.net) appropriate for your Jakarta EE version/runtime
+choice (this sample assumes Java SE 21) and
+[Docker](https://docs.docker.com/get-docker/). Note,
+the [Maven Wrapper](https://maven.apache.org/wrapper/) is already included in the project, so a Maven install
+is not actually needed. You may first need to execute `chmod +x mvnw`.
 
-```bash
-git checkout -b githubuser/exercise1
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)